appveyor.yml

```diff
@@ -0,0 +1,27 @@
+# BEEP appveyor config for windows builds
+
+environment:
+  matrix:
+    - PYTHON: "C:\\Miniconda3-x64"
+      PYTHON_VERSION: "3.7"
+      PYTHON_ARCH: "64"
+      platform: x64
+      STANDALONE: "FALSE"
+      CONDA_PY: "37"
+      CONDA_BUILD: "TRUE"
+      SPLIT_RUN: "FALSE"
+  PYPIPWD:
+    secure: cGsD88uzW0R371kFpYuczg==
```

Comment:
How does this security work?

Reply:
Account-specific encryption, no decryption in PRs:
https://www.appveyor.com/docs/build-configuration/#secure-variables

Reply:
I'm actually not a fan of using this build strategy anyway, I think the encryption is safe, but I'm going to remove it because I'd rather not do automated releases at this point.

Note that Travis does the same thing, except with the added bonus of encrypting the variable names as well.